templates/cinder/bin/kolla_extend_start

```diff
@@ -0,0 +1,7 @@
+#!/bin//bash
+
+if [[ -n "$HEALTHCHECK_SERVICE" ]]; then
+    echo "Cinder Healthcheck Server started"
+    # Dump heartbeat request and error output for now
+    /usr/local/bin/container-scripts/healthcheck.py $HEALTHCHECK_SERVICE &>/dev/null &
```

Comment:
Would a sidecar container be better than a background bash job?

Reply:
I'm ok with a sidecar if that's more in line with our design (which probably is).  I just felt that at some point we may want to control the number of pods and containers per pod we require.

Reply:
Would a failure on the sidecar container restart the whole pod or just the sidecar container?

Reply:
Just the sidecar container AFAIK

Reply:
(I took "failure on the sidecar container" to mean it exits with non-zero)

Reply:
Updated the code to use the sidecar healthcheck container pattern.